From 807614887ce310471b72032b4385eed9312acc23 Mon Sep 17 00:00:00 2001
From: Mark Hindley <mark@hindley.org.uk>
Date: Sat, 19 Aug 2023 20:09:52 +0100
Subject: [PATCH] man/Makefile: fix clean recipe (Closes: #1047054).

---
 man/Makefile | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/man/Makefile b/man/Makefile
index 11062419..99475bd8 100644
--- a/man/Makefile
+++ b/man/Makefile
@@ -21,8 +21,7 @@ endif
 install: all
 
 clean distclean:
-	for man in $(MANPAGES) ; do \
-	   if [ -f "$$man.orig" ] ; then mv "$$man.orig" "$$man" ; fi \
-	done
-	for lang in $(LANGUAGES) ; do rm -rf "$$lang" "$$lang.po" ; done
-
+ifdef PO4A
+	po4a $(PO4A_OPTS) --rm-translations po/po4a.cfg
+endif
+	rm -f *.po sysvinit-man.pot
-- 
2.39.2

